README.md

```diff
@@ -1,3 +1,78 @@
 # Microsoft Graph CLI
 
+## Required tools
+A commandline tool is required. We recommend:
+- [Windows Terminal + version](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701?hl=en-us&gl=us)
+- A commandline that supports Bash?
+- MacOS terminal
+- PowerShell version 7.3.2
+
+## Target project requirements
+Download the CLI SDK to get started for 
+Windows, Mac, or Linux here on the assets link of the latest release: [CLI SDK Download](https://github.com/microsoftgraph/msgraph-cli/releases)
+
+
+## Registering an application in Azure AD
+
+## Creating an application registration
+
+> **Note:** this step is required if your client will be calling APIs that are protected by the Microsoft Identity Platform like Microsoft Graph.
+
+Follow the instructions in [Register an application for Microsoft identity platform authentication](register-app.md) to get an application ID (also know as a client ID).
+
+## Creating the client application
```

Comment:
What does this mean? Creating client application to me sounds like it's talking about creating an Azure AD client application. I think it should read: Logging in